mainapp/templates/mainapp/request_form.html

```diff
@@ -29,19 +41,17 @@ <h3 id="myModalLabel">Allow Location - നിങ്ങളുടെ സ്ഥല
         <button class="btn" data-dismiss="modal" id="modal_ok_button" aria-hidden="true">Ok</button>
       </div>
     </div>
-  </div>
-
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+	</div>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/lodash@4.17.10/lodash.min.js" integrity="sha256-/GKyJ0BQJD8c8UYgf7ziBrs/QgcikS7Fv/SaArgBcEI=" crossorigin="anonymous"></script>
+	<!-- <script src="http://www.openlayers.org/api/OpenLayers.js"></script> -->
+	<script src="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.1.3/build/ol.js"></script>
```

Comment:
Consider loading this later or only if user clicks the button. Defer loading this JS.